<details><summary>Failed to submit graph for 6a0ac74ec7a27fd09d62c86042d7d8e111e3ed13
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmpi2ju2ybf/flash-attn-feedstock.yaml --custom-metadata-file /tmp/tmpi2ju2ybf/custom_metadata.yaml --release-name flash-attn-feedstock-3-6a0ac74ec7a27fd09d62c86042d7d8e111e3ed13 flash-attn-feedstock/recipe', with error: 'INFO:root:Found feedstock flash-attn-feedstock at /tmp/tmpi2ju2ybf/flash-attn-feedstock/recipe
INFO:root:Build/Test graph from flash-attn-feedstock
Created by sbx_user1051 on 2026-04-17 17:53:09
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmpi2ju2ybf/flash-attn-feedstock.yaml --custom-metadata-file /tmp/tmpi2ju2ybf/custom_metadata.yaml --release-name flash-attn-feedstock-3-6a0ac74ec7a27fd09d62c86042d7d8e111e3ed13 flash-attn-feedstock/recipe

INFO:root:Found abs.yaml in /tmp/tmpi2ju2ybf/flash-attn-feedstock
INFO:root:Creating graph from feedstock at: flash-attn-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 416, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 643, in from_feedstock
    build_configs, test_configs = expand_build_matrix(
                                  ~~~~~~~~~~~~~~~~~~~^
        feedstock.recipe_dir_path, src_channels, parsed_user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 82, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels, user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 527, in variants_for_feedstock_and_platform
    variants = get_package_variants(parsed.recipe, config, variants=user_variants)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/variants.py", line 701, in get_package_variants
    combined_spec, specs = get_package_combined_spec(
                           ~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipedir_or_metadata, config=config, variants=variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/variants.py", line 669, in get_package_combined_spec
    combined_spec = combine_specs(specs, log_output=config.verbose)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/variants.py", line 382, in combine_specs
    values = _combine_spec_dictionaries(
        specs, extend_keys=extend_keys, zip_keys=zip_keys, log_output=log_output
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/variants.py", line 317, in _combine_spec_dictionaries
    raise ValueError(
    ...<4 lines>...
    )
ValueError: All entries associated by a zip_key field must be the same length.  In /tmp/tmpi2ju2ybf/conda_build_config.yaml, cuda_compiler_version and pytorch are different (2 and 1)
'
aws lambda request id: 425e05b6-5d87-5a8c-8296-67ea283653f5
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%60425e05b6-5d87-5a8c-8296-67ea283653f5%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/recipe/conda_build_config.yaml
+++ b/recipe/conda_build_config.yaml
@@ -1,9 +1,3 @@
 cuda_compiler_version:
   - 12.8
   - 13.0
-pytorch:
-  - 2.8
-  - 2.9
-zip_keys:
-  - cuda_compiler_version
-  - pytorch

